api/cf-rpc-types/src/lp.rs

```diff
@@ -135,3 +135,9 @@ impl TryFrom<CloseOrderJson> for CloseOrder {
 		})
 	}
 }
+
+#[derive(Serialize, Deserialize, Clone)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, Clone, Debug)]
```

This is nice to have...